hack/dockerfile/install-binaries.sh

```diff
@@ -24,7 +26,16 @@ do
 			git clone https://github.com/opencontainers/runc.git "$GOPATH/src/github.com/opencontainers/runc"
 			cd "$GOPATH/src/github.com/opencontainers/runc"
 			git checkout -q "$RUNC_COMMIT"
-			make static BUILDTAGS="seccomp apparmor selinux"
+			make static BUILDTAGS="$RUNC_BUILDTAGS"
+			cp runc /usr/local/bin/docker-runc
+			;;
+
+		runc-dynamic)
+			echo "Install runc version $RUNC_COMMIT"
+			git clone https://github.com/opencontainers/runc.git "$GOPATH/src/github.com/opencontainers/runc"
+			cd "$GOPATH/src/github.com/opencontainers/runc"
+			git checkout -q "$RUNC_COMMIT"
+			make BUILDTAGS="$RUNC_BUILDTAGS"
```

Comment:
Since this is the only difference between `runc` and `runc-dynamic` case, can we merge them together and have a `$rule` variable (name up to you) that is either empty or hold `static` according to the value of `$prog`?

If yes, same can be done for `containerd` and `containerd-dynamic` below.